src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
All the route classes will have annotations in them for each method, it's just the \info (and maybe tags) don't really go with just one class. And currently the way this is laid out i can just call something like ```swagger src/ -o path/to/output``` and it'll go through src/ and get all the annotations. As far as I know that's the easiest way to do it with swagger-php